modules/siddhi-core/src/test/java/org/wso2/siddhi/core/query/window/LenghtBatchWindowTestCase.java

```diff
@@ -94,7 +94,7 @@ public void LengthBatchWindowTest2() throws InterruptedException {
                 "@info(name = 'query1') " +
                 "from cseEventStream#window.lengthBatch(" + length + ") " +
                 "select symbol,price,volume " +
-                "insert into outputStream ;";
+                "insert expired events into outputStream ;";
 
```

Comment:
Not really. I was trying to understand how length batch window  behaves.
Forgot to undo the change. Should I change this and resend pull request?

On Thu, Jun 16, 2016 at 10:36 AM, Mohanadarshan Vivekanandalingam <
notifications@github.com> wrote:

> In
> modules/siddhi-core/src/test/java/org/wso2/siddhi/core/query/window/LenghtBatchWindowTestCase.java
> <https://github.com/wso2/siddhi/pull/163#discussion_r67287826>:
>
> > @@ -94,7 +94,7 @@ public void LengthBatchWindowTest2() throws InterruptedException {
> >                  "@info(name = 'query1') " +
> >                  "from cseEventStream#window.lengthBatch(" + length + ") " +
> >                  "select symbol,price,volume " +
> > -                "insert into outputStream ;";
> > +                "insert expired events into outputStream ;";
> >
>
> Any reason for changing Siddhi core test case ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wso2/siddhi/pull/163/files/1461d4d148096e5993d3d1111a6b916b8e5d0f72#r67287826>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARvsJzM1ys93ancdKd6L6Vw93l34rcnUks5qMNnTgaJpZM4I3AzG>
> .
>



-- 
Charini Vimansha Nanayakkara
Software Engineer at WSO2
Mobile: 0714126293